src/Microsoft.SqlTools.Hosting/Microsoft.SqlTools.Hosting.csproj

```diff
@@ -26,10 +26,10 @@
 		<InternalsVisibleTo Include="Microsoft.SqlTools.ServiceLayer.UnitTests" />
 		<InternalsVisibleTo Include="Microsoft.SqlTools.ServiceLayer.IntegrationTests" />
 		<InternalsVisibleTo Include="Microsoft.SqlTools.ServiceLayer.Test.Common" />
-        <!-- Workaround for SRGen issues -->
+		<!-- Workaround for SRGen issues -->
```

Comment:
what srgen issues? It could be easy to fix.

Reply:
*shrug* Been here forever https://github.com/microsoft/sqltoolsservice/pull/321

Probably something we can remove, I'll do a quick check after this PR to see if there's any issues.